### PR TITLE
Add Skill Creator and Skill Wizard

### LINKED ISSUE #2381: Create a Skill Creator Wrap Page
**Actual Behaviour**
Currently we want to implement draft feature for skill too. For that first we need to have a skill creator wrap page similar to Bot Builder wrap.

**Expected Behaviour**
Implement a skill creator wrap page similar to BotBuilderWrap page.

**Screenshots of the issue**
The page will look similar to this.
Screenshot 2019-06-26 at 11 27 44 PM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2381 

Changes: Skill creator wrap shows all the skills created by the user and draft skills.

Demo Link: https://pr-2386-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-06-28 at 7 57 07 AM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -200,6 +201,11 @@ class App extends Component {
                 path="/skills/skillCreator"
                 component={SkillCreator}
               />
+              <ProtectedRoute
+                exact
+                path="/skills/skillCreatorWrap"
```
Comment (akshatnitd) [inline src/App.js]: Think of a better route name.

File: src/components/cms/SkillCreator/SkillCreatorWrap.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+                  size="small"
+                  style={{
+                    boxShadow:
+                      'rgba(0, 0, 0, 0.12) 0rem 0.063rem 0.375rem, rgba(0, 0, 0, 0.12) 0rem 0.063rem 0.25rem',
+                  }}
+                >
+                  <Add
+                    style={{
+                      height: '2.5rem',
+                    }}
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillCreatorWrap.js]: Loading state doesn't show up.

File: src/components/cms/SkillCreator/SkillCreatorWrap.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+                  />
+                </Fab>
+                <CardContent>Create a new skill</CardContent>
+              </SkillCard>
+            </Link>
+            {this.showSkills()}
+          </SkillCardWrap>
+          <br />
+          <h2>Drafts</h2>
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillCreatorWrap.js]: Display an empty text when length of drafts is 0.

**Comment by harsh-jindal** [comment]
Looks like there is some issue with some images. Looking into it.

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Fixed. Also I have renamed skill creator(currently on susi.ai) to skill wizard and named skill creator to the current page in screenshot in description

**Pull request merged**